OpenRA.Mods.Common/Traits/Infantry/ScaredyCat.cs

```diff
@@ -81,13 +84,13 @@ void INotifyIdle.TickIdle(Actor self)
 
 		void INotifyDamage.Damaged(Actor self, AttackInfo e)
 		{
-			if (e.Damage.Value > 0)
+			if (e.Damage.Value > 0 && self.World.SharedRandom.Next(100) <= info.PanicChance)
 				Panic();
 		}
 
 		void INotifyAttack.Attacking(Actor self, Target target, Armament a, Barrel barrel)
 		{
-			if (self.World.SharedRandom.Next(100) < info.AttackPanicChance)
+			if (self.World.SharedRandom.Next(100) <= info.AttackPanicChance)
```

Comment:
`<=` is wrong here - this would mean that a panic chance of 0 would have a 1% chance of triggering.